sway-fmt-v2/src/items/item_enum.rs

```diff
@@ -79,26 +79,30 @@ impl Format for ItemEnum {
 impl CurlyBrace for ItemEnum {
     fn open_curly_brace(line: &mut String, formatter: &mut Formatter) {
         let brace_style = formatter.config.items.item_brace_style;
+        let extra_width = formatter.config.whitespace.tab_spaces;
         let mut shape = formatter.shape;
+        let open_brace = Delimiter::Brace.as_open_char();
         match brace_style {
             ItemBraceStyle::AlwaysNextLine => {
                 // Add openning brace to the next line.
-                line.push_str("\n{\n");
-                shape = shape.block_indent(1);
+                line.push_str(&format!("\n{}\n", open_brace));
```

Comment:
A nicer alternative to using `format!` when appending to an existing `String` (or any buffer, including `stdout`/`stderr`!) is to use `write!` or `writeln!`, e.g.

```suggestion
                writeln!(line, "\n{}", open_brace);
```